web/packages/teleport/src/Bots/Add/GitHubActions/ConnectGitHub.tsx

```diff
@@ -103,8 +110,12 @@ export function ConnectGitHub({ nextStep, prevStep }: FlowStepProps) {
 
     const hosts = new Set<string>();
     repoRules.forEach(rule => {
-      const { host } = parseRepoAddress(rule.repoAddress);
-      hosts.add(host);
+      try {
+        const { host } = parseRepoAddress(rule.repoAddress);
+        hosts.add(host);
+      } catch (err) {
+        setHostError(() => IVNALID_HOST_ERROR({ rule: rule.repoAddress, error: err }))
```

Comment:
i'm curious if this actually works? b/c this looks like setting a function that'll need to be called later. it looks more accurate to do this but 🤷‍♀️ :

> setHostError(getInvalidHostErrorComp({ rule: rule.repoAddress, error: err }))

also i just noticed we shouldn't be all capping functions 😅 (at least to my knowledge it's only all cap for hard coded consts)

Reply:
Yup, those were just string constants at some point, but I changed them to components and forgot to rename them.

I've renamed those and moved them to the bottom of the file, as we usually leave the main component as the first item. TY.